lib/fs/dune

```diff
@@ -1,5 +1,6 @@
 (library
  (name fs)
  (libraries
+  str
```

Comment:
Lol, I was sincerely surprised to learn that OCaml doesn't have a string search function, and one need to use regular expressions.

Oh, well.